api/taskcluster/webhook_test.go

```diff
@@ -130,6 +130,9 @@ func TestCreateAllRuns_success(t *testing.T) {
 
 	mockC := gomock.NewController(t)
 	suitesAPI := checks.NewMockSuitesAPI(mockC)
+	suitesAPI.EXPECT().GetSuitesForSHA(sha).Return([]shared.CheckSuite{
+		shared.CheckSuite{SHA: sha},
+	}, nil)
 	suitesAPI.EXPECT().PendingCheckRun(sha, "chrome")
```

Comment:
Hmm, how did this pass? `PendingCheckRun` now takes a suite. Did you forget to regenerate the mock?

Reply:
Actually, it takes an array of suites to update... so "this is fine". I'll update the test :)